From 46ff8cb75446b22e0a4d97e1815ef31efa151182 Mon Sep 17 00:00:00 2001
From: Quintin Hill <Quintin.Hill@arqiva.com>
Date: Thu, 13 Apr 2017 13:06:12 +0100
Subject: [PATCH 6/6] condor_utils: fix build with newer Visual Studio

It seems timezone is no longer an alias of _timezone by default.
---
 src/condor_utils/misc_utils.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/condor_utils/misc_utils.cpp b/src/condor_utils/misc_utils.cpp
index f95f03d16d..537a4b2596 100644
--- a/src/condor_utils/misc_utils.cpp
+++ b/src/condor_utils/misc_utils.cpp
@@ -117,6 +117,9 @@ my_timezone(int isdst)
 	  return tzname[0];
   }
 #else
+#ifndef timezone
+#define timezone _timezone
+#endif
 	// On Win32, tzname is useless.  It return a string like
 	// "Central Standard Time", which doesn't follow any standard,
 	// and thus cannot be used in things like valid SQL statements.
-- 
2.13.5

